app/frontend/components/domains/step-code/checklist/energy-step-code-compliance/compliance-grid/energy-step.tsx

```diff
@@ -16,9 +16,7 @@ export const EnergyStep = function EnergyStep({ checklist }: IProps) {
     <>
       <GridRowHeader>{t(`${translationPrefix}.step`)}</GridRowHeader>
       <GridData>
-        <TextFormControl
-          inputProps={{ isDisabled: true, textAlign: "center", value: checklist.proposedEnergyStep || "-" }}
-        />
+        <TextFormControl inputProps={{ isDisabled: true, textAlign: "center", value: checklist.requiredEnergyStep }} />
```

Comment:
this isDisabled does not seem to be working.  It does not save it, but the dropdown can till be pressed?